on ln. 29, +0.0
	of __init__(...)
	| bound successfully to localhost:58357
on ln. 41, +7.02
	of handle_connections(...)
	| received connection from 127.0.0.1
on ln. 170, +9.03
	of handle_http_requests(...)
	| [127.0.0.1:34644] received 'GET / HTTP'
on ln. 50, +16.79
	of handle_connections(...)
	| user-interrupt caught
	| fatal error, exiting ...
on ln. 59, +16.81
	of __del__(...)
	| garbage collecting SocketServer instance
